src/app/page.tsx

```diff
@@ -1,12 +1,15 @@
-import { type NextPage } from "next";
-import { SEO } from "~/components/SEO";
-import { Button, Text } from "@mantine/core";
+import { type Metadata } from "next";
 import Link from "next/link";
+import Button from "~/clients/components/Button";
+import Text from "~/clients/components/Text";
 
-const Home: NextPage = () => {
+export const metadata: Metadata = {
+  title: "Intro | app name",
+  description: "Generated by create-t3-app",
+};
+const HomePage = () => {
   return (
     <>
-      <SEO title="Intro" />
       <main className="flex h-screen flex-col items-center justify-center">
```

Comment:
I wrote this line temporarily and I was wondering if we use both Tailwind and Mantine together, or can we style it only with Mantine?

Reply:
In my opinion, we can mainly use Mantine as much as possible. 
We can set up some sharing layout for both pages later with Mantine.